src/main/java/baritone/pathing/movement/CalculationContext.java

```diff
@@ -136,7 +136,9 @@ public Block getBlock(int x, int y, int z) {
     }
 
     public double costOfPlacingAt(int x, int y, int z, IBlockState current) {
-        if (!hasThrowaway) { // only true if allowPlace is true, see constructor
+        if (!hasThrowaway || baritone.getPathingBehavior().isCloseToHome()) { /** only true if allowPlace is true, see constructor
```

Comment:
You should either cache this (like `hasThrowaway`) so it can't change during pathing, or make it actually use the x,y,z parameters to depend on the block being checked instead of the player position

Reply:
Make it so that it checks the xyz, not player position. Baritone would currently mine towards home, and then after entering home, would have to recalculate path, or even may just continue on the current path.

Reply:
ih yeah, that really has to be fixed asap

Reply:
Changed in latest commit